### PR TITLE
Add the README.md content

### PR DESCRIPTION
**Pull request by imobachgs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Getting the Sources
+===================
+
+To get the source code, clone the GitHub repository:
+
+    $ git clone https://github.com/yast/yast-caasp.git
+
+If you want to contribute into the project you can
+[fork](https://help.github.com/articles/fork-a-repo/) the repository and clone your fork.
```
Comment (lslezak) [inline README.md:27]: Please add [`CONTRIBUTING.md`](https://github.com/yast/yast-yast2/blob/master/CONTRIBUTING.md) and refer to it here.

**Review by lslezak — CHANGES_REQUESTED** [review]
Add CONTRIBUTING.md

File: README.md (hunk 1 above)
Comment (imobachgs) [reply README.md:27]: The `CONTRIBUTING.md` file is added in the other https://github.com/yast/yast-caasp/pull/1. About referencing it here, I just followed the same approach we have in the README of other yast2 projects (like yast2 or yast2-installation).

Comment (lslezak) [reply README.md:27]: Oh, then it's OK...

**Review by lslezak — APPROVED** [review]

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**